### ISSUE TITLE
Oomox plugin request.

### ISSUE THREAD

**Issue by brokoli-mate** [vinceliuice/Canta-theme#48]
Can you please create plugin for oomox-gui to easily build your theme in different colors with the gui?

**Comment by ProkopHapala** [comment]
Yes, I would also like that. I want o rather adjust border and margins to use less space on my laptop.
